File: 은지/LV2/피보나치 수.js
function solution(n) {
  var a = 0
  var b = 1
  for (let i=0; i<n-1; i++) {
    var c = a+b
    a = b % 1234567
    b = c % 1234567
  }
  var answer = b
  return answer;
}

File: 은지/LV2/피보나치 수.py
def fibo(x):
    if x == 0:
        return 0
    elif x == 1:
        return 1
    else:
        return solution(x-1) + solution(x-2)

def solution(n):
    return fibo(n) % 1234567


###############################################


def solution(n):
    a = 0
    b = 1
    for _ in range(n-1):
        c = a + b
        a = b
        b = c
    answer = b % 1234567
    return answer